<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawToolbarWidget</class>
 <widget class="QWidget" name="DrawToolbarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>32</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Рисование</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="lineLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="textLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="frontLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="polygonLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="pixmapLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="cursorLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="undoLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="redoLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="groupLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
